File: samples/src-js/sample3.js
function validate(num) {
  if (num < 1 || num > 9) {
    console.log('Out of range');
  } else if (num !== parseInt(num)) {
    console.log('Not an integer');
  } else {
    console.log('Just right');
    return true;
  }
  return false;
}

console.log(validate(-5));
console.log(validate(15));
console.log(validate(5.2));
console.log(validate(5));

File: src-py/sample3.py
def validate(num):
  if (num < 1 or num > 9):
    print('Out of range')
  elif (num != int(num)):
    print('Not an integer')
  else:
    print('Just right')
    return True

  return False

print(validate(-5))
print(validate(15))
print(validate(5.2))
print(validate(5))